website/content/en/docs/building-operators/golang/migration.md

```diff
@@ -128,6 +150,46 @@ type Memcached struct {...}
 type MemcachedList struct {...}
 ```
 
+## Webhooks
+
+From the SDK version `1.0.0`, webhooks are supported by the CLI. If you don't have any webhooks, you can skip this section. However, if have been use it via customizations done in your project then, you should use the tool to re-scaffold the webhooks. 
+
+A webhook can only be scaffold for a pre-existent API in your project. Then, for each case you will run the command `operator-sdk create webhook` informing the `--group`, `--kind` and `version` of the API which should be used. 
+
+The valid types are: `defaulting`, `validation` and `conversion`. Use the same type used before to do this scaffold. To create defaulting and validating webhooks :
+
+```sh
+operator-sdk create webhook \
+    --group=cache \
+    --version=<version> \
+    --kind=<Kind> \
+    --defaulting \
+    --programmatic-validation
+```
+
+And then, to create conversion webhook use:
+
+```sh
+operator-sdk create webhook \
+    --group=cache \
+    --version=<version> \
+    --kind=<Kind> \
+    --conversion 
+```
+
+After generate the webhook you will need to 
+copy the webhook definition and content from your old project to the new one. You will find the file in `api/v1/<kind>_webhook.go`. 
+
+### How to keep using `apiextensions.k8s.io/v1beta1` for Webhooks?
+
+From now on, the Webhooks that will be created by SDK using by default the Kubernetes API version `admissionregistration.k8s.io/v1` instead of `admissionregistration.k8s.io/v1beta1` and the `cert-manager.io/v1` instead of `cert-manager.io/v1alpha2`. 
```

Comment:
```suggestion
Hereafter, the webhooks that are created by SDK will use Kubernetes API version `admissionregistration.k8s.io/v1` by default instead of `admissionregistration.k8s.io/v1beta1` and `cert-manager.io/v1` instead of `cert-manager.io/v1alpha2`. 
```